internal/pkg/otel/configtranslate/otelconfig_test.go

```diff
@@ -277,6 +277,16 @@ func TestGetOtelConfig(t *testing.T) {
 			},
 		}
 	}
+
+	getBeatMonitoringConfig := func(_, _ string) map[string]any {
+		return map[string]any{
+			"http": map[string]any{
+				"enabled": true,
+				"host":    "localhost:9999",
```

Comment:
I don't think this will change anything, but shouldn't this be:
```golang
"http": map[string]any{
        "enabled": true
        "host": "localhost"
        "port": 9999
}
```
I think the port is supposed to be it's own entry in the map.  https://www.elastic.co/docs/reference/beats/metricbeat/http-endpoint

Reply:
Yeah, that's a bit misleading. Agent actually never sets the port, because we use unix domain sockets, but I'll correct it for clarity.